### DISCUSSION TITLE
Pre-Release v1.1.0alpha0 code

### DISCUSSION THREAD

**Discussion by abraunegg** [bpozdena/OneDriveGUI#128] [General]
In regards to https://github.com/bpozdena/OneDriveGUI/tree/v1.1.0-alpha0 - it might be worth updating this to be pointing at 'alpha-3' - https://github.com/abraunegg/onedrive/discussions/2521

I am in the process of updating a few more items, then will be looking to fix the logging aspect.

Generally, 'alpha-3' is very solid from testing thus far - so it would be great to start getting wider feedback.

**Comment by bpozdena** [comment]
It's hard to keep up with your rapid development 😉.  But o did test mostly on your alpha3 code. I'll push the updated description once I get to my PC.

I could only update the [AppImage pre-release notes ](https://github.com/bpozdena/OneDriveGUI/releases/tag/v1.1.0alpha0) so far.
